cowamm/kpi/competitors/uni_swap_style/largest_uni_style_kpis_4304295.sql

```diff
@@ -0,0 +1,75 @@
+-- Computes volume, tvl and APR for Uni Swap style pools (Uni, Pancake, Sushi)
+-- APR is measured as the fees earned per $ invested, over the last 24 hours, projected over 1 year
+-- The type of pool (Uni/Pancake/Sushi) is not specified
```

Comment:
shouldn't we allow for this though (we might now based on the factory contract that emitted the event). I'm pretty sure we need to know which type of pool we are talking about in order to properly render competitor information in the UI.

Ideally the result would contain the amm type or allow filtering for a specific amm type (ie. by passing in the correct factory contract address)